pkg/oomstore/push_processor.go

```diff
@@ -150,15 +147,12 @@ func (p *PushProcessor) pushToOffline(ctx context.Context, s *OomStore, groupID
 }
 
 func (s *OomStore) newRevisionForStream(ctx context.Context, groupID int, revision int64) error {
-	features, err := s.metadata.ListFeature(ctx, metadata.ListFeatureOpt{
+	features := s.metadata.ListCachedFeature(ctx, metadata.ListCachedFeatureOpt{
 		GroupID: &groupID,
 	})
-	if err != nil {
-		return err
-	}
 	entity := features[0].Entity()
```

Comment:
Is it possible for the length of features to be 0?

Reply:
not likely, otherwise user cannot push feature values

Reply:
lgtm!